tools/codegen/cmd/mixgenproc/BUILD

```diff
@@ -0,0 +1,14 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I think istio.io/mixer/pkg and istio.io/mixer/cmd might not be the final home for this stuff. I am planning to write down a doc on how we will have a protoc plugin and how it will play with codegeneration of proc interface and registrar etc. For now I think we can defer this discussion for a week or two. Once we are clear on how all the various codegen will tie together, we will automatically converge to a happy home for all this stuff.

This current structure within tools/codegen/cmd and tools/codegen/pkg.. makes it easy to move stuff later.